### DISCUSSION TITLE
Info Screen Needs Updating

### DISCUSSION THREAD

**Discussion by Cal-LF** [Jyers/Marlin#1077] [General Discussion]
@Jyers The printer Info screen still has the old Jyres Extensible_UI title as does the VS Code compile screen.
No big deal but needs to be updated to the new Jyres_UI title.

![Jyres](https://user-images.githubusercontent.com/76499230/123562973-e3f16a80-d77f-11eb-8afb-62778909efad.jpg)
![Image01](https://user-images.githubusercontent.com/76499230/123563129-f5874200-d780-11eb-8137-fc26183258bc.jpg)

**Comment by adam3654** [comment]
Lol I just noticed that today and fixed it, should I make a pull request? It was some pretty heavy lifting, but I muddled my way through it.

**Reply by Jyers** [reply]
Don't worry about it, not really worth a PR. Just got remember where that string define was 😅

**Comment by smooreace** [comment]
You might want to explain what you did so we can tell you whats wrong?

**Reply by Titan7976** [reply]
I managed to sort it out. I went back into the file and went and found the troubleshooting and changed the sd card allocation to 4K and it worked. Thank you though.
